docker/builder/build.sh

```diff
@@ -3,7 +3,7 @@
 #ccache -s
 mkdir -p /server/build_docker
 cd /server/build_docker
-cmake -G Ninja /server -DENABLE_TESTS=1
+cmake -G Ninja /server -DENABLE_TESTS=1 -DCMAKE_C_COMPILER=`which gcc-9` -DCMAKE_CXX_COMPILER=`which g++-9`
```

Comment:
Probably it is better to link `gcc-9` and `g++-9` to `gcc` and `g++` in the container, btw it is up to @alexey-milovidov